content/02.introduction.md

```diff
@@ -24,10 +24,10 @@ Prior studies have identified univariate correlations of CpG site methylation [@
 Other relevant past work includes linking point mutations and copy number variants (CNVs) with changes in methylation and expression at individual genes [@doi:10.1093/bioinformatics/btr019; @doi:10.1093/bib/bbw037] and identifying functional modules that are perturbed by somatic mutations [@doi:10.1093/bioinformatics/btq182; @doi:10.1038/ncomms9554].
 However, direct comparison among different data types for this application is lacking, particularly in the multivariate case where we consider changes to -omics-derived gene signatures rather than individual genes in isolation.
 
-We select a collection of potential cancer drivers with varying functions and roles in cancer development [@doi:10.1126/science.1235122].
+We select a collection of potential cancer drivers with varying functions and roles in cancer development.
 We use mutation status in these genes as labels to train classifiers, using each of the data types listed as training data, in a pan-cancer setting; we follow similar methods to the elastic net logistic regression approach described in Way et al. 2018 [@doi:10.1016/j.celrep.2018.03.046] and Way et al. 2020 [@doi:10.1186/s13059-020-02021-3].
-We show that there is considerable predictive signal for many genes relative to a random baseline and that gene expression and DNA methylation generally provide the best predictions of mutation state.
-Surprisingly, we find that across a variety of genes, gene expression and DNA methylation are approximately equally effective predictors.
+We show that there is considerable predictive signal for many genes relative to a random baseline, and that gene expression tends to provide good predictions of mutation state across most genes.
```

Comment:
```suggestion
We show that there is considerable predictive signal for many genes relative to a cancer-type corrected baseline and that gene expression tends to provide good predictions of mutation state across most genes.
```
I wonder if this was creating some confusion? That came through a bit in the prediction of normal.